yarn.lock

```diff
@@ -353,6 +353,11 @@ doctrine@^3.0.0:
   dependencies:
     esutils "^2.0.2"
 
+dotenv@^7.0.0:
```

Comment:
@chirag-shinde 
This has silently slipped in.
Make sure you do "yarn remove package-name" instead of deleting lines from package.json